Gemfile

```diff
@@ -25,6 +25,7 @@ gem "stimulus-rails"
 
 # Use Tailwind CSS [https://github.com/rails/tailwindcss-rails]
 gem "tailwindcss-rails"
+gem "sassc-rails"
```

Comment:
Add sass rials gem to allow for proper use of sass in this project. Sass has more features than css alone.